src/OrchardCore.Modules/OrchardCore.ContentTypes/Editors/ContentTypeSettingsDisplayDriver.cs

```diff
@@ -20,6 +20,8 @@ public override IDisplayResult Edit(ContentTypeDefinition contentTypeDefinition)
                 model.Versionable = settings.Versionable;
                 model.Securable = settings.Securable;
                 model.Stereotype = settings.Stereotype;
+                model.IsFullText = settings.IsFullText;
```

Comment:
It should be a custom driver and a custom settings file

Reply:
done